demo/services/mockWorkspaceFlowRunsApi.ts

```diff
@@ -62,4 +66,4 @@ export class MockWorkspaceFlowRunsApi extends MockApi implements IWorkspaceFlowR
     return await this.flowRuns.delete(flowRunId)
   }
 
-}
\ No newline at end of file
+}
```

Comment:
@stackoverfloweth I feel like I've been seeing this diff come up a lot. Guessing there's a linting rule or a vscode workspace setting for newlines at end of file?

Reply:
🤷 if there is, it has to be from @typescript-eslint/recommended

Reply:
we could also disable that rule with a nested .eslintrc in the demo folder

Reply:
might be a good idea. not worried about it, just curious

Reply:
That might be me actually. My last spot we always had newlines at the end of files, so it's a habit of mine now

Reply:
I don't really care which, but it needs to be configured for everyone via a workspace setting or a linting rule so that way we're not seeing these in PRs. 

Something we can figure out. Don't worry about it.